src/Magnum/BulletIntegration/DebugDraw.h

```diff
@@ -32,6 +32,7 @@
 
 #include <vector>
 #include <Corrade/Containers/ArrayView.h>
+#include <Corrade/Utility/Macros.h>
```

Comment:
Oh. I guess this slipped through in the previous PR... The parameter names don't need to be marked with `CORRADE_UNUSED` in function *declaration* in the header -- it will not generate any compiler warning. So then this include is also not needed.